client/lib/schema.js

```diff
@@ -0,0 +1,42 @@
+
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+const schema = new SimpleSchema({
+  dateRange: {
+    type: Object
+  },
+  "dateRange.start": {
+    type: Date
+  },
+  "dateRange.stop": {
+    type: Date
+  },
+  timeRange: {
+    type: Object
+  },
+  "timeRange.start": {
+    type: Date,
+    custom: function () {
+      if (this.value > this.field('timeRange.stop').value) {
+        console.log('inside sart validator', this.value);
+        return "startRangeError";
+      }
+    }
+  },
+  "timeRange.stop": {
+    type: Date,
+    custom: function () {
+      if (this.value < this.field('timeRange.start').value) {
+        console.log('inside end validator', this.value);
+        return "stopRangeError";
+      }
+    }
+  },
+});
+
+schema.messages({
```

Comment:
It is a different approach and works, and in fact should be in the code for double checking
But the task is about a component, thats why schema.js and app.js are given.

You can solve this in the component, disabling not allowed hours.

But is ok, are easy fixes which solve with better communication

Reply:
okay @cesarve77. Could you please let me know whether should I the validations into the component ?

Reply:
You can use the followings properties of TimePicker to disabled not allowed times 

disabledHours | to specify the hours that cannot be selected 
disabledMinutes | to specify the minutes that cannot be selected 
disabledSeconds | to specify the seconds that cannot be selected

Reply:
@cesarve77, I have disabled the end time based on the start time selected in the time picker.